### DISCUSSION TITLE
Support for Fantia product page

### DISCUSSION THREAD

**Discussion by ian040504** [KJHJason/Cultured-Downloader#161] [Q&A] [answered]
Is Cultured-Downloader-CLI support for preview content on Fantia product page?

**Comment by KJHJason** [comment, marked as answer]
Nope but **Preview** content?
Do you mean that you would want to download all the product thumbnails or product demo contents instead of the paid product item/files?

It should be possible to implement  it but I have yet to look into this so you could make a feature request first in the issue page.

**Reply by ian040504** [reply]
Yes, I mean those preview content, including thumbnails or demo videos.
I'll open a feature request in the issue page.
